pegasus/sites.v3/hourofcode.com/public/highlights.haml

```diff
@@ -0,0 +1,45 @@
+---
+title: Highlights
+layout: wide
+---
+
+%link{href: "/css/highlights.css", rel: "stylesheet", id: "highlight-styles"}
+%link{href: "/css/highlights-mobile.css", rel: "stylesheet", id: "highlight-styles-mobile"}
+
+%h1 Hour of Code Highlights
+%h3 Some of our favorite moments of the Hour of Code campaign.
+
+.highlights-content
+  - tag = 'highlight'
+  -DB[:cdo_news_items].where(tag_s:tag).limit(100).each do |item|
```

Comment:
also have you tested with 100 entries to make sure the page scales okay to that number?